cmake/introspect_llvm.d

```diff
@@ -109,11 +109,11 @@ string llvmVersion() {
     return "LLVM_" ~ parts.joiner("_").toUTF8;
 }
 
-string llvmCppFlags() {
-    const flags = execute([llvmCmd, "--cppflags"]);
+string llvmCxxFlags() {
+    const flags = execute([llvmCmd, "--cxxflags"]);
 
     // -std=c++0x is required to run on travis.
-    return flags.output.strip ~ " -std=c++0x -fno-exceptions -fno-rtti";
+    return flags.output.strip ~ " -std=c++0x -finline-functions -fno-strict-aliasing -fno-exceptions -fno-rtti";
```

Comment:
Should be OK to use cxxflags and probably what should be done because the c++ code that exists is mainly to interact with clang.

But remove `-std=c++0x`. Why? As the comment states it where to make it work on travis. Shouldn't be an issue anymore.
Remove `-finline-functions`. Why? Unnecessary.
Remove `-fno-strict-aliasing`. Why? If it doesn't work without this flag then it is highly likely a bug in the code. The flag is a workaround.
Remove `-fno-exceptions` and `-fno-rtti`. Let llvm-config decide if these flags (via --cxxflags) should be used or not.